quetz/authorization.py

```diff
@@ -135,6 +168,9 @@ def assert_assign_user_role(self, role: Optional[str]):
             return self.assert_server_roles([SERVER_OWNER, SERVER_MAINTAINER])
 
     def assert_server_roles(self, roles: list, msg: Optional[str] = None):
+        if self.is_supertoken:
+            return "supertoken"
```

Comment:
I don't like how we're sneaking the super token into the authorization layer here. But I also understand that anything more explicit would require big changes in the Quetz authentication and authorization model.